### PR TITLE
Add Yu-Gi-Oh Saikyou Card Battle!

### PR DESCRIPTION
**Pull request by gagkog**
Adds Yu-Gi-Oh Saikyou Card Battle!.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/yu-gi-oh-saikyou-card-battle/game.dat [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title = "Yu-Gi-Oh Saikyou Card Battle!"
```
Comment (Leo626) [inline games/yu-gi-oh-saikyou-card-battle/game.dat]: Missing the `!` at the end of `Yu-Gi-Oh`.

File: games/yu-gi-oh-saikyou-card-battle/game.dat [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+description = "Yu-Gi-Oh! Card Game by Konami based on the work of  Kazuki Takahashi"
```
Comment (Leo626) [inline games/yu-gi-oh-saikyou-card-battle/game.dat]: Would flow better this:

> Yu-Gi-Oh! Saikyou Card Battle is a card game by Konami based on the work of Kazuki Takahashi.

**Comment by gagkog** [comment]
sorry I tested both thte original room and the english translation and both run exactly the same, thanks for all the changes, if i need to change something let me know also tell me where :P cause I dont now exactly where I can change things

**Comment by Leo626** [comment]
@gagkog The title is inconsistent. Looking at Gamefaq's entry, I think it should be "Yu-Gi-Oh! Duel Monsters Saikyo Card Battle!". Also add three screenshots from the original game.

Make the changes in your [repository](https://github.com/gagkog/citra-games-wiki/tree/pr-builder-yu-gi-oh-saikyou-card-battle-72/games/yu-gi-oh-saikyou-card-battle).

**Comment by gagkog** [comment]
upgraded the title and added images, I try to change the boxart for the japanese one but for some reason I couldn´t

**Comment by Leo626** [comment]
@gagkog Small nitpick with the boxart. Can you get one without the "sheen" in the logo. Like one where the logo is "plain". The boxart looks a little weird with that logo. Also, rename the screenshots to something more descriptive like title.png for the title screen. Fix those and I'll merge it.

**Comment by gagkog** [comment]
done

**Pull request merged**